test/e2e/operator.go

```diff
@@ -131,7 +100,11 @@ var _ = Describe("ARO Operator - Geneva Logging", func() {
 		}
 
 		err := wait.PollImmediate(30*time.Second, 15*time.Minute, mdsdReady)
-		Expect(err).NotTo(HaveOccurred())
```

Comment:
don't drop this line or the tests will start passing when it should have failed.